src/main/java/Tasks/TaskType.java

```diff
@@ -0,0 +1,8 @@
+package duke.task;
+
+public enum TaskType {
```

Comment:
Perhaps can consider removing this enum class, does not seems like it is being used.

Reply:
Also, you can think about javaDoc as well here.